scripts/joinmarket-qt.py

```diff
@@ -1690,6 +1707,7 @@ def selectWallet(self, testnet_seed=None):
             #ignore return value as there is no decryption failure possible
             self.loadWalletFromBlockchain(firstarg, pwd)
 
+
```

Comment:
I don't think we need this?

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Right, I vaguely remembered that rule. Never liked it personally, but anyway: this doesn't follow that rule. This is a method function inside a class so should be separated by one line, not two, according to that?

Reply:
Yes, looked at code more carefully, that blank line shouldn't be there. Will remove that too.